src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -324,6 +334,14 @@ private boolean isDuplicateBarcodeError(String errorMessage) {
     return errorMessage.contains(BARCODE_ALREADY_EXISTS);
   }
 
+  private boolean isUserTypeNotPopulatedError(String errorMessage) {
+    return errorMessage.contains(USER_TYPE_NOT_POPULATED);
+  }
+
+  private boolean isUserTypeNotPopulatedError(AsyncResult<Response> reply) {
+    return isDesiredError(reply, ".*user type was not populated.*");
+  }
```

Comment:
lets move these methods after 2 method related to barcodes(and not between them)